docs/src/Groups/autgroup.md

```diff
@@ -11,8 +11,8 @@ automorphism_group(G::GAPGroup)
 
 The following functions are available for automorphisms, some of them similar to the corresponding functions for homomorphisms of groups.
 ```@docs
-is_invariant(f::GAPGroupElem{AutomorphismGroup{T}}, H::T) where T<:GAPGroup
-restrict_automorphism(f::GAPGroupElem{AutomorphismGroup{T}}, H::T, A=automorphism_group(H)) where T <: GAPGroup
+is_invariant(f::GAPGroupElem{AutomorphismGroup{T}}, H::GAPGroup) where T <: GAPGroup
```

Comment:
The T in this line is irrelevant for the first part

Reply:
Hmm, I try to put the same line into the .md file that appears in the src file, and this seems to work.
Do we have a better convention (perhaps for the section on the `docs` folder in the developer documentation)?

Reply:
This is in fact fine, I just didn't read properly. Sorry for the confusion